Formula/c/chainloop-cli.rb

```diff
@@ -0,0 +1,28 @@
+class ChainloopCli < Formula
```

Comment:
why not just Chainloop rather than ChainloopCli?

Reply:
You use the CLI to interact with the Chainloop platform. I modeled this after the `lacework-cli` formula, which is the same sort of situation.

Reply:
maybe lacework-cli should be lacework formula as well, my initial thought was there would be some naming conflict. 

most of the formulae are CLI, so there is no need to specify `-cli` IMO

Reply:
I think it makes sense to specify CLI for formulae that are just CLI to interact with platforms. It reduces ambiguity. I don't see us renaming things like `awscli` either..

Reply:
yeah, I guess it is fine.

Reply:
personally, I still prefer chainloop over chainloop-cli, but nothing blocking merging this PR though